runtime/oti/j2sever.h

```diff
@@ -36,13 +36,13 @@
 #define J2SE_17  0x1700
 #define J2SE_18  0x1800
 #define J2SE_19  0x1900
-#define J2SE_2018_3  0xB700            /* 0xB7 is 183 which refers to Java 18.3 */
+#define J2SE_1A  0x1A00
```

Comment:
Please comment that this means Java 10

Reply:
or we could use a different constant like J2SE_v10

Reply:
Changed it to `J2SE_v10` and added a comment to indicate it is `Java 10`.